recipes/geospatial/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "geospatial" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geospatial-{{ version }}.tar.gz
+  sha256: 93edf988a6606e21f8b13251c2e8c6c073ef5b8f7b96d51ec38ef8b0a00e4587
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - geemap
+    - leafmap
+    - python >=3.7
+    - xarray
+
+test:
```

Comment:
It would be good to add a pip check here as well. Something like this one: https://github.com/conda-forge/azure-quantum-feedstock/blob/1864c7c7528cbad2af029f83de1c3ddda5267726/recipe/meta.yaml#L44-L47

Reply:
I think I added the `pip check` before, but the checks failed and I am not sure why. I just re-added `pip check` to see if it can pass the checks this time.

Reply:
I have fixed all `pip check` errors except `pointpats 2.2.0 requires opencv-contrib-python, which is not installed.` However, `pointpats` is ready listed in the recipe and it has been installed. See Line 1276 docker build. I am not sure which package depends on `pointpats`. I also added `libopencv`, `opencv`, and `py-opencv`, but it still does not solve the issue. 
Should this issue be resolved by the `pointpats` package on the conda-forge channel? How could have`pointpats` passed checks if it is missing some required packages?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=484880&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1276

Reply:
The issue has been resolved after removing the `pysal` package from the reqs. All checks have passes and it is ready for your review. Thanks.